### PR TITLE
Upgrade @polkadot/api version to 14

### PR DESCRIPTION
**Pull request by stepanLav**
# Description
That PR upgrade polkadot/api library in order to handle the latest changes, deployed on westend.
Error:
```
2024-11-21T10:28:05.969Z <fetch-#1> ERROR failed to fetch Block hash="0xa7ce8788b1deec8be4ecc77d0740999b8e1b37efe5821f076ef07befac6ac33b" height="23544613"
2024-11-21 10:28:05             VEC: Unable to decode on index 0 createType(ExtrinsicUnknown):: Unsupported unsigned extrinsic version 5
```

Tested locally:

Screenshot 2024-11-21 at 14 25 37

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have tested locally
- [x] I have performed a self review of my changes
- [x] Updated any relevant documentation
- [x] Linked to any relevant issues
- [x] I have added tests relevant to my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code is up to date with the base branch
- [x] I have updated relevant changelogs. [We suggest using chan](https://github.com/geut/chan/tree/main/packages/chan)

### CODE REVIEW

File: packages/node/CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.3.0] - 2024-11-21
+### Changed
+- Update polkadot/api library to 14 version
```
Comment (stwiname) [inline packages/node/CHANGELOG.md:11]: This should be in the unreleased section

File: packages/types/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/types",
-  "version": "3.11.3",
+  "version": "3.12.0",
```
Comment (stwiname) [inline packages/types/package.json:3]: Can you please revert this.

File: packages/types/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.12.0] - 2024-11-21
+### Added
+- Update polkadot/api library to 14 version
```
Comment (stwiname) [inline packages/types/CHANGELOG.md:11]: This should be in the unreleased section

**Review by stwiname — CHANGES_REQUESTED** [review]
There are some tests failing relating to the API service that need to be fixed

File: packages/types/CHANGELOG.md (hunk 3 above)
Comment (stepanLav) [reply packages/types/CHANGELOG.md:9]: I see, fixed

File: packages/node/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/node",
-  "version": "5.2.10-1",
+  "version": "5.3.0",
```
Comment (yoozo) [inline packages/node/package.json:3]: Can you please revert this. You don’t need to update the `version` field in the `package.json`.

Comment (stepanLav) [reply packages/node/package.json]: ok, thanks for fix it

**Review by yoozo — APPROVED** [review]

**Comment by stwiname** [comment]
There are tests relevent to this change that are failing, https://github.com/subquery/subql/actions/runs/11968220823/job/33381688090?pr=2596#step:8:16511

**Pull request closed without merging**